htdocs/core/boxes/box_factures.php

```diff
@@ -102,10 +103,13 @@ public function loadBox($max = 5)
 			$sql .= " FROM (".MAIN_DB_PREFIX."societe as s,".MAIN_DB_PREFIX."facture as f";
 			if (!$user->rights->societe->client->voir && !$user->socid) $sql .= ", ".MAIN_DB_PREFIX."societe_commerciaux as sc";
 			$sql .= ")";
+			$sql .= " LEFT JOIN ".MAIN_DB_PREFIX."paiement_facture as pf ON f.rowid=pf.fk_facture ";
 			$sql .= " WHERE f.fk_soc = s.rowid";
 			$sql .= " AND f.entity IN (".getEntity('invoice').")";
 			if (!$user->rights->societe->client->voir && !$user->socid) $sql .= " AND s.rowid = sc.fk_soc AND sc.fk_user = ".$user->id;
 			if ($user->socid)	$sql .= " AND s.rowid = ".$user->socid;
+			$sql .= " GROUP BY s.nom, s.rowid, s.email, s.code_client, s.logo, f.ref, f.date_lim_reglement,";
+			$sql .= " f.type, f.datef, f.total, f.tva, f.total_ttc, f.paye, f.fk_statut, f.rowid";
```

Comment:
Can you check this page https://wiki.dolibarr.org/index.php?title=Language_and_development_rules#MySQL-MariaDB_specificity
And check you have enabled this option. When this option are on, it help to find bug into sql request because i think there is one error on your group by (list of fields must match all fields into select that are not calculation functions). With this option, you immediatly have a reported error that help to write cleaner sql.
Can you check.